.github/workflows/go_dvoting_test.yml

```diff
@@ -12,20 +12,20 @@ jobs:
     name: Scenario
     runs-on: ubuntu-latest
     steps:
-      - name: Use Go 1.23
-        uses: actions/setup-go@v4
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+
+      - name: Use Go 1.23.7
+        uses: actions/setup-go@v5
         with:
-          go-version: '1.23'
+          go-version: '1.23.7'
```

Comment:
I usually stick to a minor version like `1.23`, as the patch-versions usually bring good and not-breaking changes. So new patch-versions are automatically picked up.